spaces/domains.json

```
@@ -42,6 +42,7 @@
   "dao.wetu.be": "wetube.eth",
   "vote.fomodao.xyz": "fomodao.eth",
   "vote.betswirl.com": "betswirl.eth",
+  "vote.candlelabs.org": "cndl.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.candlelabs.org": "cndl.eth",
```